recipe/laravel.php

```diff
@@ -79,6 +80,15 @@ function artisan($command, $options = [])
     };
 }
 
+function artisan_path()
```

Comment:
Lets just define a new config option: `{{bin/artisan}}`.

Reply:
Yeah I suppose that would make more sense! Will push some changes soon